src/Neos.NeosIo/Resources/Private/Scss/Molecules/_Navigation.scss

```diff
@@ -115,3 +115,45 @@
 	@extend .nav--breadcrumb;
 	padding: $basic-spacing+px 0;
 }
+
+/**
+ * Language Switcher Navigation:
+ *
+ *     @example
+ *     <nav>
+ *       <ul class="nav nav--language-switcher">
+ *         <li><a href="">German</a></li>
+ *         <li><a href="">English</a></li>
+ *         <li><a href="">Spanish</a></li>
+ *       </ul>
+ *     </nav>
+ */
+.nav--language-switcher {
+	font-size: rs(14);
+	line-height: $basic-spacing+px;
+	padding: $basic-spacing+px 0;
+
+	> li {
+		&:first-child {
+			font-weight: $bold;
```

Comment:
This keeps the currently selected language from being `bold`. Always the first one is `bold` and not the one that has the `current` class.